rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java

```diff
@@ -3285,8 +3294,11 @@ void createFlyoverFederationInformation_OK() {
             signatureCache
         );
 
+        Federation genesisFederation2 = FederationTestUtils.getGenesisFederation(bridgeConstantsRegtest);
```

Comment:
Why another genesis federation? What's wrong with the one already delcared above?

Reply:
Done!

**Variable Names Issues Fixed**
1. `genesisFederation` turns into `activeFederation` since it is used by a `when` clause like an `activeFederation`.
2. `genesisFederation2` turns into `genesisFederation`.